src/main/java/billgates/usecases/insert_entry/InsertEntryPresenter.java

```diff
@@ -0,0 +1,22 @@
+package billgates.usecases.insert_entry;
+
+import billgates.interface_adapters.Updatable;
+import billgates.interface_adapters.Updatable_insert;
+
+import java.util.List;
+
+public class InsertEntryPresenter implements InsertEntryOutputPort{
+
+    private final Updatable_insert view;
+
+    public InsertEntryPresenter(Updatable_insert view) {
+        this.view = view;
+    }
+
+    @Override
+    public void insertEntry(InsertEntryResponseModel model){
```

Comment:
Similar problem in this pull request as the last one that I have reviewed [here](https://github.com/CSC207-2022F-UofT/course-project-bill-gates/pull/35).

To my knowledge, we do not need to implement a `ViewModel` or `output port` that is used to update the display or the presenter. The purpose of this `InsertEntryUseCase` is simply to insert the entry that we have obtained from the `InputPort` into the database by calling methods that is used to interact with it.